pkg/services/folder/folderimpl/unifiedstore.go

```diff
@@ -262,7 +262,7 @@ func (ss *FolderUnifiedStoreImpl) GetChildren(ctx context.Context, q folder.GetC
 			return nil, err
```

Comment:
This is the main issue here (in line 260) that we'll call Get folder for each folder returned in the list